Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let version = "1"
```

Comment:
This should be version `v3.2.1` to match the commit sha which it was built from.